problem/templates/problem/edit.html

```diff
@@ -1,108 +1,115 @@
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+{% extends 'problem/base.html' %} 
+{% block body_block %} {% load staticfiles %}
+<script src='http://cdn.ckeditor.com/4.4.6/standard/ckeditor.js'></script>
+<script src="{% static 'problem/js/edit.js' %}"></script>
+<div class="container" id="Body">
+  <input type="hidden" name="owner" value="{{ user.pk }}">
+  <div style="width:80%; float:left;">
+    <ul class="nav nav-tabs" role="tablist">
+      <li class="active">
+        <a href="#info" role="tab" data-toggle="tab">Info</a>
+      </li>
+      <li>
+        <a href="#description" role="tab" data-toggle="tab">Description</a>
+      </li>
+      <li>
+        <a href="#sampio" role="tab" data-toggle="tab">Sample IO</a>
+      </li>
+      <li>
+        <a href="#tag" role="tab" data-toggle="tab">Tag</a>
+      </li>
+      <li>
+        <a href="#testcase" role="tab" data-toggle="tab">TestCase</a>
+      </li>
+    </ul>
+
+    <style>
+    .tab-pane {
+      padding-top: 10px;
+    }
+    </style>
+    <div class="tab-content">
+      <form method="POST" action='' enctype="multipart/form-data">
+        <div>
+          <a class='btn btn-success' target='_blank' id='preview_button'>Preview</a>
+          <input type="submit" value="Save" class="btn btn-primary" 
+            style="width:100px" href="{% url 'problem:problem.views.problem' %}">
+        </div>
+        {% csrf_token %}
+        <div class="tab-pane active" id="info">
+          {% if owner %}
+          <input type='hidden' name='owner' value='{{owner}}'>
+          {% endif %}
+          {% include 'problem/editTab/info.html' with form=form %}
+        </div>
+        <div class="tab-pane" id="description">
+          <strong>Description</strong>
+          <textarea cols="80" rows="20" id="des" name="description">{{description}}</textarea>
+          <br><strong>Input</strong>
+          <textarea cols="80" rows="20" id="inText" name="input_description">{{input}}</textarea>
+          <br><strong>Output</strong>
+          <textarea cols="80" rows="20" id="outText" name="output_description">{{output}}</textarea>
+
+          <script>
+          CKEDITOR.replace('des')
+          CKEDITOR.replace('inText')
+          CKEDITOR.replace('outText')
+          </script>
+        </div>
+        <div class="tab-pane" id="sampio">
+          <div class='panel panel-default' style='float:left;width:47%'>
+            <div class='panel-heading'>Sample Input</div>
+            <textarea name='sample_input' class='panel-body' 
```

Comment:
"sample_input" and next line "sample_in" may name consistently, it is just a suggestion maybe others have different idea.

p.s. same as "sample_input" below